REWARD_Ontology/.htaccess

```diff
@@ -0,0 +1,5 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://www.cap-a.eu/model/reward.ttl/$1 [R=302,L]
```

Comment:
You don't have any pattern that would be used in that `$1` substitution.  Also looks non-standard to add something after a ttl file name like that.  What are you trying to do there?  I'd recommend testing rules locally if you can.

Reply:
My intention is to forward the url: https://w3id.org/REWARD_Ontology to https://www.cap-a.eu/model/reward.ttl.

Reply:
According to the last commit I think that now it should be OK. thanks.